lib/fidlib/fidlib.c

```diff
@@ -2294,7 +2294,7 @@ grabWord(char **pp, char *buf, int buflen) {
 	     (*q != ',' && *q != ';' && *q != ')' && *q != ']' && *q != '}'))
 	 q++;
    }
-   len= q-p;
+   len= (int)(q-p);
```

Comment:
Instead of casting to (int) everywhere, its probably better to use size_t instead. Do we need negative values? Can the cast from signed to unsigned overflow?

Reply:
That will involve a bigger refactoring, because all structures storing length are int too. 
size_t changes the length on different platform which brings a risk of regression.

Reply:
Fair Point.